skfp/preprocessing/filters/rule_of_veber.py

```diff
@@ -0,0 +1,82 @@
+from typing import Union
+
+from rdkit.Chem import Mol, rdMolDescriptors
+
+from skfp.bases.base_filter import BaseFilter
+
+
+class RuleOfVeber(BaseFilter):
+    """
+    Rule of Veber.
+
+    The rule is designed to identify molecules likely to exhibit good oral
+    bioavailability, based on the number of rotatable bonds and the topological
+    polar surface area (TPSA) [1]_.
+
+    Molecule must fulfill conditions:
+
+    - number of rotatable bonds <= 10
+    - TPSA <= 140
+
+    Parameters
+    ----------
+    allow_one_violation : bool, default=False
+        Whether to allow violating one of the rules for a molecule. This makes the
+        filter less restrictive.
+
+    n_jobs : int, default=None
+        The number of jobs to run in parallel. :meth:`transform_x_y` and
+        :meth:`transform` are parallelized over the input molecules. ``None`` means 1
+        unless in a :obj:`joblib.parallel_backend` context. ``-1`` means using all
+        processors. See Scikit-learn documentation on ``n_jobs`` for more details.
+
+    batch_size : int, default=None
+        Number of inputs processed in each batch. ``None`` divides input data into
+        equal-sized parts, as many as ``n_jobs``.
+
+    verbose : int, default=0
+        Controls the verbosity when generating conformers.
+
+    References
+    -----------
+    .. [1] `Veber, D.F, Johnson S. R., Cheng H., Smith B. R., Ward K. W. , Kopple K. D.
+        "Molecular properties that influence the oral bioavailability of drug candidates."
```

Comment:
Should be "Molecular Properties That Influence the Oral Bioavailability of Drug Candidates" (directly from https://pubs.acs.org/doi/10.1021/jm020017n)